ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/disconnected/techpreview/provision/cucushift-installer-rehearse-vsphere-ipi-disconnected-techpreview-provision-chain.yaml

```diff
@@ -19,5 +19,8 @@ chain:
   - name: BASTION_HOST_SUBNET
     default: "qe-segment"
     documentation: the subnet to provision bastion host
+  - name: "STEPUP_IMAGE_REGISTRY_WITH_PVC"
```

Comment:
Suggest to enable this option in a non-techpreview install workflow, e.g: `vsphere-ipi-disconnected`

Reply:
@jianlinliu  thanks review, updated it according your comments.